### ISSUE TITLE
When test on 'Synthetic_v3_InstanceSegmentation' test dataset, break out Segmentation fault (core dumped) without any more imformation

### ISSUE THREAD

**Issue by TobyZhouWei** [VinAIResearch/ISBNet#29]
Hi, I am working to test on 'Synthetic_v3_InstanceSegmentation' test dataset. However, no matter how to split data to small blocks, it awalys come out "Segmentation fault (core dumped)" without any more imformation to fix it. what can I do to reslove [it?]

(isbnet) root@autodl-container-690911b33c-4a0c7e92:~/ISBNet-master# python tools/test.py configs/stpls3d/isbnet_stpls3d.yaml head_stpls3d_val.pth --out result/test
2023-07-16 10:37:20,083 - INFO - Load state dict from head_stpls3d_val.pth
2023-07-16 10:37:20,229 - INFO - unexpected key in source state_dict: criterion.empty_weight
2023-07-16 10:37:20,233 - INFO - Load test dataset: 100 scans
Segmentation fault (core dumped)

**Comment by huanghuang113** [comment]
> Hi, I am working to test on 'Synthetic_v3_InstanceSegmentation' test dataset. However, no matter how to split data to small blocks, it awalys come out "Segmentation fault (core dumped)" without any more imformation to fix it. what can I do to reslove [it?]
> 
> (isbnet) root@autodl-container-690911b33c-4a0c7e92:~/ISBNet-master# python tools/test.py configs/stpls3d/isbnet_stpls3d.yaml head_stpls3d_val.pth --out result/test 2023-07-16 10:37:20,083 - INFO - Load state dict from head_stpls3d_val.pth 2023-07-16 10:37:20,229 - INFO - unexpected key in source state_dict: criterion.empty_weight 2023-07-16 10:37:20,233 - INFO - Load test dataset: 100 scans Segmentation fault (core dumped)

Hi, have you solved the problem of testing on the stpls3d26-28 dataset, I trained the whole model and I don't know how to test on unlabeled data now!

**Comment by huanghuang113** [comment]
> 



> 
Hello, thank you for your answer, last time under Mask3D also you answered my question, I looked at your homepage, it seems that you are also Chinese, I still have doubts about this issue, may I trouble you, add your contact information to consult you? Thank you very much! My wechat is: 17343247483. if you feel inconvenient, it does not matter, anyway, thank you very much for your reply!I will try your method, thanks again!

**Comment by huanghuang113** [comment]
> Hi, you need to add label for val not for test. The easiest way is to add 2 columns, like this x,y,z,r,g,b,100, 100.

I apologize for my rude request, what you mean is that when I executed test.py, the files inside my val folder were semantically labeled and instance labeled, and the txt file for the test inside the test folder, simply had xyzrgb, and I executed the task python tools/test.py . /configs/plant/isbnet_stpls3d_test.yaml ... /work_dirs/stpls3d/exp200/latest.pth
--out
result/stpls3d , Since my input test data doesn't have sem_label and ins_label, he is just wrong in calculating the AP, and when running the program, I debugged it and found that the pred_insts are not simply a list of only ins_label, please ask you to implement test.py's for the test file to output their prediction results yet, thank you very much for your answer!

**Comment by TobyZhouWei** [comment]
Hi, maybe I need to explain. I did not read the code carefully so I did not kown the reason. The solution is the result of trying luck. I have not work on the code yet, so sorry that I can not help you more.
Best!

**Comment by WS517** [comment]
> > 嗨，我正在努力在“Synthetic_v3_InstanceSegmentation”测试数据集上进行测试。但是，无论如何将数据拆分为小块，它都会出现“分段错误（核心转储）”，而没有任何更多信息来修复它。我能做些什么来重新爱上它？
> > （isbnet） root@autodl-container-690911b33c-4a0c7e92：~/ISBNet-master# python tools/test.py configs/stpls3d/isbnet_stpls3d.yaml head_stpls3d_val.pth --out result/test 2023-07-16 10：37：20,083 - INFO - 来自 head_stpls3d_val.pth 的加载状态字典 2023-07-16 10：37：20,229 - INFO - 源 state_dict 中的意外键：criterion.empty_weight 2023-07-16 10：37：20,233 - INFO - 负载测试数据集：100 次扫描 分段故障（核心转储）
> 
> 嗨，你有没有解决在stpls3d26-28数据集上测试的问题，我训练了整个模型，我现在不知道如何在未标记的数据上测试！

Hello and thanks for your great work. I have trained the stpls3d dataset using the code to get the model, I would like to know how to test the stpls3d unlabeled test data, which has only 6 columns of data, which are xyzrgb, could you please take the time to answer my question, I would appreciate if you can help me with my problem!
